src/cozmo/pets.py

```diff
@@ -23,7 +23,7 @@
 observable events whenever the pet is observed, etc.
 
 If a pet goes off-screen, it will be assigned a new object_id (and
-therefore Pet object) when it returns. This is because the system can
+therefore a new Pet object will be created) when it returns. This is because the system can
```

Comment:
This makes that one line noticeably longer than the others here - please adjust this (and the subsequent 2 lines) so that they're all about 80 characters wide - e.g. add a line break after "because".
Note: slightly over 80 is fine - e.g. don't add a whole new line if it's 82 characters.